tests/test_common.cc

```diff
@@ -11,9 +11,9 @@
 #include "accelerator/common_core.h"
 #include "iota_api_mock.hh"
 
+using ::testing::_;
```

Comment:
Why do you move this line? Is it because of CI?

Reply:
It is a typo, I will recover it.

Reply:
Oh I misread. It was modified by /hook/formatter.